### PR TITLE
Move VS Code plugin, add IntelliJ, add APIsecurity.io newsletter

### PR DESCRIPTION
**Pull request by DSotnikov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,6 +300,7 @@ Contributions are most welcome. Categories are also open to suggestions!
 - [JSON-LD](https://json-ld.org/)
 
 ## Learning Resources
+- [APIsecurity.io weekly newsletter and encyclopedia on API Security](https://apisecurity.io)
```
Comment (yosriady) [inline README.md:303]: Can we move this to the bottom of the list?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ Contributions are most welcome. Categories are also open to suggestions!
 - [Swagger Inspector](https://swagger.io/tools/swagger-inspector/): Test and auto-generate OpenAPI documentation for any API.
 - [Swagger Editor](http://editor.swagger.io/): An editor for designing Swagger specifications.
 - [Swagger Tools and Integrations](https://swagger.io/open-source-integrations/): A list of libraries and frameworks serving the Swagger ecosystem.
+- [OpenAPI extension for Microsoft Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi): Microsoft VS Code IDE has an OpenAPI extension that includes OpenAPI / Swagger file creation, HTML preview, editing, navigation, code snippets, IntelliSense, security static analysis.
```
Comment (yosriady) [inline README.md:39]:
```suggestion
- [OpenAPI extension for VS Code](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi):  Visual Studio Code (VS Code) extension that provides support for the OpenAPI Specification.
```

File: README.md [hunk 3]
```diff
@@ -36,6 +36,8 @@ Contributions are most welcome. Categories are also open to suggestions!
...
+- [OpenAPI extension for JetBrains IDEs: IntelliJ, PyCharm, etc](https://plugins.jetbrains.com/plugin/14837-openapi-swagger-editor): This IDE plugin for JetBrains includes OpenAPI / Swagger file creation, HTML preview, editing, navigation, code snippets, IntelliSense, security static analysis.
```
Comment (yosriady) [inline README.md:40]:
```suggestion
- [OpenAPI plugin for JetBrains IDEs](https://plugins.jetbrains.com/plugin/14837-openapi-swagger-editor): Jetbrains plugin that provides support for the OpenAPI Specification.
```

**Review by yosriady — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DSotnikov) [reply README.md:303]: Sure, feel free to order as you see fit

**Pull request merged**